exe/kubernetes-deploy

```diff
@@ -25,6 +26,7 @@ ARGV.options do |opts|
   opts.on("--allow-protected-ns") { allow_protected_ns = true }
   opts.on("--no-prune") { prune = false }
   opts.on("--template-dir=DIR") { |v| template_dir = v }
+  opts.on("--verbose-log-tags") { verbose_log_tags = true }
```

Comment:
This is for core to use to turn on the inclusion of `[context][namespace]` on all log lines. Suggestions for better flag name appreciated.